services/crypto_create.proto

```diff
@@ -151,4 +151,19 @@ message CryptoCreateTransactionBody {
      * If true, the account declines receiving a staking reward. The default value is false.
      */
     bool decline_reward = 17;
+
+    /**
+     * The bytes to be used as the account's alias. The bytes will be 1 of 2 options. It will be the 
+     * serialization of a protobuf Key message for an ED25519/ECDSA_SECP256K1 primitive key type. If the 
+     * account is ECDSA_SECP256K1 based it may also be the public address, calcluated as the last 20 bytes of
+     * the keccak-256 hash of the ECDSA_SECP256K1 primitive key. Currently only primitive key bytes are 
+     * supported as the key for an account with an alias. ThresholdKey, KeyList, ContractID, and
+     * delegatable_contract_id are not supported.
+     *
+     * At most only one account can ever have a given alias on the network.
```

Comment:
The "can _ever_ have" may be confusing; we will only enforce uniqueness relative to aliases _currently_ on the network.

Reply:
Fixed